en/docs/chapter_searching/index.md

```diff
@@ -4,6 +4,6 @@
 
 !!! abstract
 
-    Searching is an unknown adventure, where we may need to traverse every corner of a mysterious space, or perhaps quickly pinpoint our target.
+    Searching is an adventure into the unknown; where we may need to traverse every corner of a mysterious space, or perhaps we’ll quickly pinpoint our target.
     
-    In this journey of discovery, each exploration may yield an unexpected answer.
+    On this journey of discovery, each exploration may return an unexpected answer.
```

Comment:
"search" sounds more relevant in this context comparing to "exploration" as it has a clear target rather than going around without specific purpose. "end up with" sounds more natural than "return" here.

Reply:
Hi Thomas
thanks for the suggestions.
I wasn't sure about the exploration part. I was thinking of trying to balance between technical and creating the imagery in the picture given in the index section, changing to search does make sense, but it may degrade the imagery a bit I think. So I'm not sure if I should change it or not.

thanks